pkg/agent/containerd/runtimes.go

```diff
@@ -16,13 +17,13 @@ import (
 // The key holds the name of the runtime
 type runtimeConfigs map[string]templates.ContainerdRuntimeConfig
 
-// findContainerRuntimes returns a list of container runtimes that
+// grepRuntimes returns a list of container runtimes that
 // are available on the system. It checks install locations provided via
 // the potentialRuntimes variable.
 // The binaries are searched at the locations specivied by locationsToCheck.
 // Note: check the given locations in order.
 // The given fs.FS should represent the filesystem root directory to search in.
-func findContainerRuntimes(root fs.FS, potentialRuntimes runtimeConfigs, locationsToCheck []string, foundRuntimes runtimeConfigs) {
+func grepRuntimes(root fs.FS, potentialRuntimes runtimeConfigs, locationsToCheck []string, foundRuntimes runtimeConfigs) {
```

Comment:
I don't like the use of the name `grep` we aren't using grep to find things. 
```suggestion
func searchForRuntimes(root fs.FS, potentialRuntimes runtimeConfigs, locationsToCheck []string, foundRuntimes runtimeConfigs) {
```

The comment should also be rewritten to differentiate it from the new `findContainerRuntimes` because the sentence
"returns a list of container runtimes that are available on the system" sure sounds confusing now.